Tests/NumLib/LocalToGlobalIndexMap.cpp

```diff
@@ -107,12 +107,41 @@ TEST_F(NumLibLocalToGlobalIndexMapTest, DISABLED_SubsetByComponent)
         new MeshLib::MeshSubsets{selected_subset.get()}};
 
     auto dof_map_subset = std::unique_ptr<NumLib::LocalToGlobalIndexMap>{
-        dof_map->deriveBoundaryConstrainedMap(0,  // variable id
-                                              1,  // component id
+        dof_map->deriveBoundaryConstrainedMap(1,  // variable id
+                                              0,  // component id
                                               std::move(selected_component),
                                               some_elements)};
 
     // There must be as many rows as nodes in the input times the number of
     // components.
     ASSERT_EQ(selected_nodes.size(), dof_map_subset->dofSizeWithGhosts());
 }
+
+#ifndef USE_PETSC
+TEST_F(NumLibLocalToGlobalIndexMapTest, MultipleVariablesMultipleComponents)
+#else
+TEST_F(NumLibLocalToGlobalIndexMapTest, DISABLED_MultipleVariablesMultipleComponents)
+#endif
+{
+    // test 2 variables (1st variable with 1 component, 2nd variable with 2 components)
```

Comment:
Didn't that already work with the unfixed DOF table? (I think @endJunction said so, if I remember correctly)
1st var 2 comp, 2nd var 1 comp didn't work, did it?